### ISSUE TITLE
@Title and @Narrative use the same format in html reports

### ISSUE THREAD

**Issue by dxxvi** [renatoathaydes/spock-reports#88]
In the generated html, they are both under the same 
```
<pre class="narrative">
  This is the title
  This is the multi-line
  narrative.
</pre>
```
It'll be great if they have different format.

**Comment by renatoathaydes** [comment]
Hi @dxxvi. You're right, it would be nicer to be able to format the two differently, I will have a look.

**Comment by renatoathaydes** [comment]
Fixed in version 1.3.0.
